env/dev/env/config.cljs

```diff
@@ -1,7 +1,5 @@
 (ns env.config)
 
```

Comment:
This file doesn't need to be commited, not sure actually why is not in `gitignore`, does anyone know?

Reply:
@cammellos this got committed as it was modified by `lein cljfmt fix`. Should I remove it from PR?